diff --git a/lib/makeWebpackConfig.js b/lib/makeWebpackConfig.js
index 9a200f7593158576db2d2cc0410826eef26eb2e6..c901bdf3200b33d3b59971e217cb5da91bf80ec6 100644
--- a/lib/makeWebpackConfig.js
+++ b/lib/makeWebpackConfig.js
@@ -161,7 +161,6 @@ module.exports = async (playroomConfig, options) => {
         chunksSortMode: 'none',
         chunks: ['preview'],
         filename: 'preview/index.html',
-        publicPath: '../',
       }),
       ...(options.production ? [] : [new FriendlyErrorsWebpackPlugin()]),
     ],
